service/static/index.html

```diff
@@ -114,10 +114,12 @@ <h4>Address Information:</h4>
             <div class="form-horizontal">
               <div class="form-group">
                 <div class="col-sm-offset-2 col-sm-10">
-                  <button type="submit" class="btn btn-primary" id="addr-search-btn">Search for Customer Addresses</button>
+                  <button type="submit" class="btn btn-primary" id="addr-query-btn">List A Customer' Addresses</button>
```

Comment:
Wondering if we should change this button to Search for Customer Address and the button currently labeled as Search for Customer Address to "Retrieve Address", I think if the button labels are kept similar to the way they are for the customer portion of the UI, it makes it easier to figure out how to use

Reply:
no problem